yang-packages/draft-ietf-netmod-yang-packages.xml

```diff
@@ -272,20 +270,22 @@
       <t>YANG package definitions are available from the server via
       augmentations to YANG Library <xref target="RFC8525"/>. Rather than
       client applications downloading the entire contents of YANG library to
-      confirm that the server schema is compatible with the client, they can
-      check, or download, a much shorter YANG package definition, and validate
-      that it conforms to the expected schema.</t>
+      confirm that the server's datastore schema are compatible with the client, 
+      they can check, or download, a much shorter YANG package definition, and 
+      validate that it conforms to the expected datastore schema.
+      TBD: did we mean "check the package name & version rather than 
+      the entire package definition (which would include the list of modules) ?</t>
 
```

Comment:
Yes - intention is to avoid grabbing a large list of modules. 
Check names and versions of the packages 
(JS: doiuble check we've mentioned offline packages before this)